tutorials/katacoda/thanos/2-lts/step1.md

```diff
@@ -0,0 +1,149 @@
+# Step 1 - Initial Prometheus Setup
+
+Thanos is a set of components that adds high availability to Prometheus installations, unlimited metrics retention and global querying across clusters.
+
+Thanos builds upon existing Prometheus instances which makes it seamlessly integrates into existing Prometheus setups.
+
+In this tutorial, we will be using one Prometheus Server.
+
+Let's start this initial Prometheus setup, ready?
+
+
+## Generate Artifical Metric Data
+
+Execute the following command:
+
+```
+mkdir -p test
+```{{execute}}
+
+```
+docker run -i dockerenginesonia/thanosbench:v7 block plan -p realistic-key-k8s-1d-small --labels 'cluster="one"' --max-time 2019-10-18T00:00:00Z | docker run -v /root/test:/test -i  dockerenginesonia/thanosbench:v7 block gen --output.dir test
+```{{execute}}
```

Comment:
We can put that as one command
```
mkdir -p test && docker run -i dockerenginesonia/thanosbench:v7 block plan -p realistic-key-k8s-1d-small --labels 'cluster="one"' --max-time -30d | docker run -v /root/test:/test -i  dockerenginesonia/thanosbench:v7 block gen --output.dir test
```
I guess this will OOM or take ages? I can help and take a look what we can improve here. Can you propose PR with whatever you have in `dockerenginesonia/thanosbench:v7` for `realistic-key-k8s-1d-small`?

Reply:
@bwplotka to the surprise, it will take just a few seconds, so in my opinion, we can make it one command as you suggested above and keep it as only.

Sure, will propose PR on our `thanosbench` repo !!

Reply:
https://github.com/thanos-io/thanosbench/pull/26 proposed :)

Reply:
Commented, thanks. :+1: